src/framework/theme/components/tooltip/tooltip.directive.ts

```diff
@@ -61,14 +61,24 @@ import { NbTooltipComponent } from './tooltip.component';
  * It is also possible to specify tooltip color using `nbTooltipStatus` property:
  * @stacked-example(Colored Tooltips, tooltip/tooltip-colors.component)
  *
+ * Tooltip has a number of triggers which provides an ability to show and hide the component in different ways:
+ *
+ * - Click mode shows the component when a user clicks on the host element and hides when the user clicks
+ * somewhere on the document outside the component.
+ * - Hint provides capability show$ the component when the user hovers over the host element
+ * and hide$ when user hovers out of the host.
+ * - Hover works like hint mode with one exception - when the user moves mouse from host element to
+ * the container element the component remains open, so that it is possible to interact with it content.
+ * - Focus mode is applied when user focuses the element.
+ * - Noop mode - the component won't react to the user interaction.
  */
 @Directive({ selector: '[nbTooltip]' })
 export class NbTooltipDirective implements AfterViewInit, OnDestroy {
 
   context: Object = {};
 
   /**
-   * Popover content which will be rendered in NbTooltipComponent.
+   * Tooltip content which will be rendered in NbTooltipComponent.
    * Available content: template ref, component and any primitive.
```

Comment:
I don't think it is a correct statement. `NbTooltipDirective` may accept only `string` content: https://github.com/akveo/nebular/pull/1138/files#diff-d0e689a1ded55212ebd0a0202c9dc95bR86